terraform/modules/web/variables.tf

```diff
@@ -15,9 +15,7 @@ variable "enabled" {
 
 variable "logging_bucket" {
   description = "Object representing the logging S3 bucket."
-  type = object({
-    arn = string
-  })
+  type        = string
 }
```

Comment:
I think we can drop the `logging_bucket` variable and instead reference `platform.splunk_logging_bucket`.